ice_setup/ice.py

```diff
@@ -1074,17 +1098,21 @@ def parse_args(self):
         sudo_check()
 
         if parser.has('all'):
-            repo_path = parser.get('all')
-            configure_local(repo_path=repo_path)
-            configure_remotes(repo_path=repo_path)
+            package_path = parser.get('all')
+            configure_local('calamari-server', package_path or CWD)
```

Comment:
seems that the pattern of `package_path or CWD` is very repetitive across calls.

Since `CWD` is a global variable already, wouldn't it be better to just pass `package_path` and do the `package_path or CWD` in the `configure_*` functions?